gadget/position.go

```diff
@@ -92,7 +92,7 @@ type PositionedContent struct {
 
 func (p PositionedContent) String() string {
 	if p.Image != "" {
-		return fmtIndexAndName(p.Index, fmt.Sprintf("%v@%#x", p.Image, p.StartOffset))
+		return fmtIndexAndName(p.Index, fmt.Sprintf("%v@%#x:%v", p.Image, p.StartOffset, p.Size))
```

Comment:
doesn't the ":" make one think  of start and end, instead of start and size? maybe use "@off{size}"  ?